### ISSUE TITLE
uninitialized constant Carto::StaticMapsURLHelper

### ISSUE THREAD

**Issue by alonsogarciapablo** [CartoDB/cartodb#9007]
I got an `uninitialized constant Carto::StaticMapsURLHelper` error when loading an embed.

Here's the stack trace:

```
NameError in Carto/builder/public/embeds#show

Showing /home/ubuntu/www/production.cartodb.com/current/app/views/carto/builder/public/embeds/show.html.erb where line #5 raised:

uninitialized constant Carto::StaticMapsURLHelper

Extracted source (around line #5):

2: 
3: <%= content_for(:description) do %><%= @vizjson[:description] ? "#{@vizjson[:description].strip_tags}. " : "" %>Map created by <%= @vizjson[:user][:fullname] %> in CARTO<% end %>
4: 
5: <%= content_for(:twitter_card) do %><%= Carto::StaticMapsURLHelper.new.url_for_static_map(request, @visualization, 560, 300) %><% end %>
6: 
7: <%= content_for(:facebook_card) do %><%= Carto::StaticMapsURLHelper.new.url_for_static_map(request, @visualization, 1200, 630) %><% end %>
8: 
Rails.root: /home/ubuntu/www/production.cartodb.com/current

Application Trace | Framework Trace | Full Trace
app/views/carto/builder/public/embeds/show.html.erb:5:in `block in _app_views_carto_builder_public_embeds_show_html_erb__3689997915482980958_35946260'
app/views/carto/builder/public/embeds/show.html.erb:5:in `_app_views_carto_builder_public_embeds_show_html_erb__3689997915482980958_35946260'
app/controllers/carto/builder/public/embeds_controller.rb:20:in `show'
app/controllers/application_controller.rb:99:in `wrap_in_profiler'
```

As discussed offline, requiring that dependency in the template explicitly with `<% require_dependency 'static_maps_url_helper' %>` fixes the problem. We might need to change it in other templates too.

cc: @juanignaciosl @javitonino

**Comment by alonsogarciapablo** [comment]
Closing this since I didn't see it anymore.

**Comment by juanignaciosl** [comment]
I got one today and fixed all at #9145.
